pkg/skaffold/build/cache/retrieve.go

```diff
@@ -40,6 +41,9 @@ func (c *cache) Build(ctx context.Context, out io.Writer, tags tag.ImageTags, ar
 
 	start := time.Now()
 
+	ctx, endTrace := instrumentation.StartTrace(ctx, "pkg/skaffold/runner", "CheckCache")
```

Comment:
WDYT?
```suggestion
	ctx, endTrace := instrumentation.StartTrace(ctx, "pkg/skaffold/build/cache", "CheckBuildCache")
```

Reply:
`CheckBuildCache` is more readable to me.  Done.